src/MSBuildTaskHost/MSBuildTaskHost.csproj

```diff
@@ -1,9 +1,5 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
-  <!-- The .NET Core version of MSBuild doesn't support targeting .NET Framework 3.5.  So in that case, we import
-       a .props file that prevents building the project from doing much of anything. -->
```

Comment:
This comment is no longer _entirely_ true but it appears to be true enough for this project because of resources :(